bin/check_samplesheet.py

```diff
@@ -47,7 +47,7 @@
 while True:
     line = fin.readline()
     if line:
-        lspl = [x.strip() for x in line.strip().split(',')]
+        lspl = [x.strip() for x in line.strip().split(',')][:-1]
```

Comment:
Is there any reason why it only checks until annotation?

Reply:
This variable is for the Stringtie and featureCounts inputs, and both processes do not need the condition information.